browser/src/Services/Configuration/DefaultConfiguration.ts

```diff
@@ -106,6 +106,9 @@ const BaseConfiguration: IConfigurationValues = {
     "tabs.height": "2.5em",
     "tabs.maxWidth": "30em",
     "tabs.wrap": false,
+
+    "ui.fontFamily": "BlinkMacSystemFont, 'Lucide Grande', 'Segoe UI', Ubuntu, Cantarell, sans-seriff",
```

Comment:
@bryphe Just noticed not, perhaps `s/Lucide/Lucida` ?

Reply:
@badosu - wow, nice catch, thank you! I'll update this.

Reply:
I am sorry @bryphe just noticed also Cantarell -> Cantarelli

Reply:
Thanks for checking and letting me know!

I looked at this one and I believe `Cantarell` is actually correct: https://en.wikipedia.org/wiki/Cantarell_(typeface)

LMK though if I'm mistaken here